pkg/apis/openebs.io/v1alpha1/cstorvolume.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
Okay, this looks better already.  As we now have actual messages defined for specifics requests, thanks